ui/src/main/resources/AzureAD/AzureADClientConfiguration.xml

```diff
@@ -0,0 +1,760 @@
+<?xml version="1.1" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<xwikidoc version="1.5" reference="AzureAD.AzureADClientConfiguration" locale="">
+  <web>AzureAD</web>
+  <name>AzureADClientConfiguration</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <parent>AzureAD.WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <version>1.1</version>
+  <title>AzureADClientConfiguration</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>true</hidden>
+  <content/>
+  <object>
+    <name>AzureAD.AzureADClientConfiguration</name>
```

Comment:
This should also use `EntraID`

Even for the space. I know there is already a `AzureAD` space, but are any of the pages we have in it, gonna be used in 2.x, after all changes are done? Do we want to keep it for the `WebPreferences`, is it necessary? I feel like we will keep this space only for migration purposes and it's ok to have all new pages in a separated space (`AzureAD` is about old stuff / 1.x, `EntraID` is about 2.x). Let me know if I am missing something

Reply:
Should I also modify the naming convention of the java source folders from `com.xwiki.azureoauth` to `com.xwiki.entraid`?

Reply:
Alright, but I'd leave the renaming for after I actually remove the classes, in #48.